pilot/pkg/serviceregistry/serviceentry/controller_test.go

```diff
@@ -596,6 +608,7 @@ func TestServiceDiscoveryWorkloadUpdate(t *testing.T) {
 		expectServiceInstances(t, sd, selector, 0, instances)
 		expectEvents(t, events,
 			Event{kind: "svcupdate", host: "selector.com", namespace: selector.Namespace},
+			Event{kind: "edscache", host: "selector.com", namespace: selector.Namespace},
```

Comment:
Similar comment, we do we need an EDS update here if we are going from 0 -> 0 instances? Do we need it or is it just a side effect?